### DISCUSSION TITLE
Navigation bar paths

### DISCUSSION THREAD

**Discussion by Dj0nf1** [daattali/beautiful-jekyll#1151] [Q&A]
I am new to website creation and beautiful jekyll, and I was wondering how to enable the "navbar-links" in the config.yml file to link to files that are in a specific directory via. a path, for example: "_posts/about" instead of "about". 

I have tried for a long time to figure this out, but it seems that I can only link to files that are in the main directory, and not files that are in a subdirectory - for example, in the _posts directory or in one I create myself.

I can't see in the documentation that it shouldn't be possible, but I am struggling!

Hope you can help! let me know if I am being unclear.

**Comment by daattali** [comment]
It is possible to link to any page. For posts, find the post's URL and just use that  (for example `https://mysite.com/2023-06-11-some-post`). For other pages that are nested in other directories, you need to get the URL of the page, and again use that.

This is a general jekyll question so if you're still unclear you can ask in any jekyll-based forum. Or purchase a support plan.
